sound/soc/sof/intel/hda-dai.c

```diff
@@ -153,47 +153,41 @@ static int hda_link_dma_params(struct hdac_ext_stream *stream,
 
 /* Send DAI_CONFIG IPC to the DAI that matches the dai_name and direction */
 static int hda_link_config_ipc(struct sof_intel_hda_stream *hda_stream,
-			       const char *dai_name, int channel, int dir)
+			       const char *dai_name, int channel, int dir, bool setup)
```

Comment:
You are repeating yourself in the commit message, the first and second sentence say the same thing.

Conversely the second paragraph is quite unclear.